src/rdb/rdb_raft.c

```diff
@@ -2356,6 +2378,24 @@ rdb_raft_get_compact_thres(void)
 	return i;
 }
 
+static unsigned int
+rdb_raft_get_ae_max_entries(void)
+{
+	unsigned int n = 32;
+
+	d_getenv_int("RDB_AE_MAX_ENTRIES", &n);
+	return n;
+}
+
+static size_t
+rdb_raft_get_ae_max_size(void)
+{
+	uint64_t size = (1ULL << 20);
+
+	d_getenv_uint64_t("RDB_AE_MAX_SIZE", &size);
```

Comment:
TODO: Document these two new environment variables, if reviewers agree on the overall approach.

Reply:
Done.